edi_webservice_oca/components/send.py

```diff
@@ -27,7 +32,10 @@ def __init__(self, work_context):
 
     def send(self):
         method, pargs, kwargs = self._get_call_params()
-        return self.webservice_backend.call(method, *pargs, **kwargs)
+        try:
+            return self.webservice_backend.call(method, *pargs, **kwargs)
+        except HTTPError as err:
+            raise EDIWebserviceSendHTTPException("EDI HTTP Error: %s" % err) from err
```

Comment:
We already know that this is a specific exception. I would simply use `EDIWebserviceSendHTTPException(str(err))`.
Also, you should be able to find the original exception in the catched exception via `__cause__`